docs/docs/quickstart-inference.md

```diff
@@ -0,0 +1,283 @@
+
+print("Llama 3.1 8B response:")
+print(llama_model.predict(test_question))
+print("\n" + "="*50 + "\n")
+print("DeepSeek V3 response:")
+print(deepseek_model.predict(test_question))
+```
+
+## Step 5: Evaluate model performance
+
+Evaluate how well different models perform on a Q&A task:
+
+```python
+# Create a simple dataset
+dataset = [
+    {"question": "What is 2 + 2?", "expected": "4"},
+    {"question": "What is the capital of France?", "expected": "Paris"},
+    {"question": "Name a primary color", "expected_one_of": ["red", "blue", "yellow"]},
+]
+
+# Define a scorer
+@weave.op()
+def accuracy_scorer(expected: str, output: str, expected_one_of: list[str] = None) -> dict:
+    """Score the accuracy of the model output."""
+    output_clean = output.strip().lower()
+    
+    if expected_one_of:
+        is_correct = any(option.lower() in output_clean for option in expected_one_of)
+    else:
+        is_correct = expected.lower() in output_clean
+    
+    return {"correct": is_correct, "score": 1.0 if is_correct else 0.0}
+
+# Evaluate a model
+@weave.op()
+def evaluate_model(model: InferenceModel, dataset: list[dict]) -> dict:
+    """Run evaluation on a dataset."""
+    scores = []
+    
+    for example in dataset:
+        # Get model prediction
+        output = model.predict(example["question"])
+        
+        # Score the output
+        score = accuracy_scorer(
+            expected=example.get("expected", ""),
+            output=output,
+            expected_one_of=example.get("expected_one_of")
+        )
+        scores.append(score["score"])
+    
+    return {
+        "model": model.model_name,
+        "accuracy": sum(scores) / len(scores),
+        "total_correct": sum(scores),
+        "total_examples": len(dataset)
+    }
+
+# Run evaluation
+results = evaluate_model(llama_model, dataset)
+print(f"Model: {results['model']}")
+print(f"Accuracy: {results['accuracy']:.2%}")
+print(f"Correct: {results['total_correct']}/{results['total_examples']}")
+```
+
+## Step 6: View your results in Weave
+
+After running these examples, you'll see links in your terminal. Click any link to view traces in the Weave UI.
+
+In the Weave UI, you can:
+- See a timeline of all your LLM calls
+- Inspect inputs and outputs for each operation
+- View token usage and estimated costs
+- Analyze latency and performance metrics
+- Compare different runs
+
+![Weave Traces View](guides/tools/imgs/image.png)
+
+## Available models
+
+W&B Inference provides access to many models including Llama 3.1 8B for general-purpose tasks, Llama 3.3 70B for complex reasoning, DeepSeek V3 for language processing, and Qwen3 235B for mathematical and structured reasoning. 
+
+For a complete list of available models with specifications, context windows, pricing, and usage limits, see the [Available Models section](guides/tools/inference.md#available-models) in the W&B Inference reference documentation.
```

Comment:
The first paragraph here is just asking to accumulate tech debt. Maybe avoid listing specific models.

Reply:
Yeah, removed